internal/cache/shadowdb_test.go

```diff
@@ -118,7 +119,7 @@ func TestNextShadowEntry(t *testing.T) {
 func TestNextShadowEntryNoShadow(t *testing.T) {
 	t.Parallel()
 
-	c := newCacheForTests(t, t.TempDir(), cache.WithTeardownDuration(0), cache.WithOfflineCredentialsExpiration(0))
+	c := testutils.NewCacheForTests(t, t.TempDir())
```

Comment:
This was called with the 2 options which are not in `NewCacheForTests()`, I know we don’t need here as we don’t have any entry, but the teardown duration to 0 is better.

Reply:
Yeah. My intention was to leave it as a default to the cache for tests, but I've added the option in the wrong function 🥲